connectors-home/modules/ROOT/partials/acb-java-module-partial-edit.adoc

```diff
@@ -0,0 +1,17 @@
+//Used in the configure-java-module-acb file for Java Module configuration in ACB
+
+// tag::advanced-tab-attributes[]
+. Select the *Advanced* tab to configure the following attributes: 
++
+[%header%autowidth.spread]
+|===
+| UI Field | Description | Required
+| Output MIME type | MIME type of the payload this component outputs.| No
+| Parameters | Appends a key-value pair for a reader property to the value of the Output MIME attribute. Multiple key-value pairs are allowed.| No
+| Output encoding | Encoding of the payload this component outputs. | No
```

Comment:
I wonder if there's a way to make this description more clear? I had to read it twice.
Payload encoding for this component's output? 
or maybe it just needs a "that"?